shells/chrome/src/main.js

```diff
@@ -21,9 +21,6 @@ function createPanelIfReactLoaded() {
   chrome.devtools.inspectedWindow.eval(`!!(
     Object.keys(window.__REACT_DEVTOOLS_GLOBAL_HOOK__._renderers).length || window.React
   )`, function(pageHasReact, err) {
-    if (!pageHasReact || panelCreated) {
```

Comment:
There are two cases as of now when the devtools does not display React tab
1. If React is not present
2. If React is present but the devtools extension do not have permission to access 'file://' urls

I agree that it is undesirable to show React tab for 1st case,
but for 2nd case (which behaves same as 1st, when there are no permissions) we have no clue why the tab was not displayed

That is why with this change, devtools will always display React tab.
It will show a message "**Unable to find React…**" for 1st case,
and a more specific message "**Please enable 'Allow access to file URLs' permission for the devtools extension**" for 2nd case